content/shmem_get_nbe.tex

```diff
@@ -0,0 +1,69 @@
+\apisummary{
+    The explicit non-blocking get routines provide a method for copying
+    data from a contiguous remote data object on the specified \ac{PE}
+    to the local data object.
+}
+
+\begin{apidefinition}
+
+\begin{C11synopsis}
+void shmem_get_nbe(TYPE *dest, const TYPE *source, size_t nelems, int pe, shmem_request_t *req);
+\end{C11synopsis}
+where \TYPE{} is one of the standard \ac{RMA} types specified by Table \ref{stdrmatypes}.
+
+\begin{Csynopsis}
+void shmem_<TYPENAME>_get_nbe(TYPE *dest, const TYPE *source, size_t nelems, int pe, shmem_request_t *req);
+\end{Csynopsis}
+where \TYPE{} is one of the standard \ac{RMA} types and has a corresponding \TYPENAME{} specified by Table \ref{stdrmatypes}.
+
+\begin{CsynopsisCol}
+void shmem_get<SIZE>_nbe(void *dest, const void *source, size_t  nelems, int pe, shmem_request_t *req);
+\end{CsynopsisCol}
+where \SIZE{} is one of \CONST{8, 16, 32, 64, 128}.
+
+\begin{CsynopsisCol}
+void shmem_getmem_nbe(void *dest, const void *source, size_t nelems, int pe, shmem_request_t *req);
+\end{CsynopsisCol}
+
+\begin{apiarguments}
+    \apiargument{OUT}{dest}{Local data object to be updated.}
+    \apiargument{IN}{source}{Data object on the \ac{PE} identified by \VAR{pe}
+        that contains the data to be copied.  This data object must be remotely
+        accessible.}
+    \apiargument{IN}{nelems}{Number of elements in the \dest{} and \source{}
+        arrays. \VAR{nelems} must be of type \VAR{size\_t}.}
+    \apiargument{IN}{pe}{Identifier of the remote \ac{PE}. \VAR{pe} must
+        be of type integer.}
+    \apiargument{IN/OUT}{req}{Either SHMEM\_REQUEST\_UNINITIALIZED, or a valid request object}
```

Comment:
DocEdit: Remove "Either". Unnecessary.

`SHMEM_REQUEST_UNINITIALIZED` is only used here. It isn't mentioned in the API description. What's the effect of using this library constant?

If this is a new environment variable that needs to be in the library, please add to section Library Constants.